src/main/resources/migrations.xml

```diff
@@ -4365,4 +4365,51 @@
 		<dropColumn tableName="tasks" columnName="customFieldEnum2" />
 	</changeSet>
 
+	<changeSet id="add-description-to-tasks" author="cigdemers">
+		<!-- add description field to tasks -->
+		<addColumn tableName="tasks">
+			<column name="description" type="text"/>
+		</addColumn>
+	</changeSet>
+
+	<changeSet id="alter-full_text-index-add-tasks_description" author="cigdemers" runInTransaction="false">
+		<sql>
+			<!-- Drop full-text materialized view (incl. indexes);
+                             copied from changeset alter-postgresql-fulltext -->
+			DROP MATERIALIZED VIEW mv_fts_tasks;
+
+			<!-- Drop generated full-text column from tasks;
+			     copied from changeset alter-postgresql-fulltext -->
+			ALTER TABLE tasks DROP COLUMN full_text;
+
+			<!-- Re-add generated full-text column to tasks, now including description;
+			     copied mostly from changeset alter-postgresql-fulltext -->
+			ALTER TABLE tasks
+				ADD COLUMN full_text tsvector GENERATED ALWAYS AS (
+						setweight(to_tsvector('simple', coalesce(tasks."shortName", '')), ${fts_high}) ||
+						setweight(to_tsvector(${fts_config}, coalesce(tasks."longName", ''))
+							  || to_tsvector('simple', coalesce(tasks."longName", '')), ${fts_high}) ||
+						setweight(to_tsvector(${fts_config}, coalesce(tasks."description", ''))
+							  || to_tsvector('simple', coalesce(tasks."description", '')), ${fts_low})
```

Comment:
`"description"` is unnecessarily quoted here, it is all lowercase so simply `description` is fine.